modules/ingest-user-agent/src/yamlRestTest/resources/test-regexes.yml

```diff
@@ -0,0 +1,3 @@
+user_agent_parsers:
```

Comment:
Did we remove this file from `src/test`? GitHub isn't showing this file as _moved_.

Reply:
I assumed that since it was in `test` that it was used by something in there, so I just made a copy. But, looking at the code again, I don't see any references to it outside of the yaml rest tests, so I can probably just delete the old one

Reply:
Yeah, if we removed everything else from `src/test` (which I assume we did) it can go as well.